change/@fluentui-react-message-bar-2f896b85-9363-41af-8569-e40177fbae6c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32850&commitId=c70ed5563faa97953bb467f491895e918ef90ce0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.chromium.png|1| Changed|
|Avatar Converged.badgeMask.chromium.png|12| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1167| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2107| Changed|

</details>


  <details><summary><b>MessageBar</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Intents Dark Mode.chromium.png|46| Changed|
|MessageBar.Multiline Without Actions.chromium.png|67| Changed|
|MessageBar.Intents High Contrast.chromium.png|91| Changed|
|MessageBar.Auto.chromium.png|17| Changed|
|MessageBar.Square.chromium.png|17| Changed|
|MessageBar.Intents RTL.chromium.png|67| Changed|
|MessageBar.Intents.chromium.png|67| Changed|
|MessageBar.Multiline.chromium.png|67| Changed|
|MessageBar.Multiline No Actions.chromium.png|67| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>